custom_dc/custom_data.md

```diff
@@ -116,11 +167,209 @@ geoId/06,2021,555,666
 geoId/08,2021,10,10
 ```
 
-## Write the data config file
+### Write the JSON config file
+
+You must define a `config.json` in the top-level directory where your CSV files are located. With the implicit schema method, you need to provide 3 specifications:
+- The input files location and entity type
+- The sources and provenances of the data
+- Optionally, additional properties of the statistical variables you've used in the CSV files
+
+Here is an example of how the config file would look for WHO CSV file we defined earlier. More details are below.
+
+```json
+{
+  "inputFiles": {
+    "adult_cig_smoking.csv": {
+      "entityType": "Country",
+      "provenance": "UN_WHO",
```

Comment:
If the columns can be in any order, should we specify which columns have entity and dates?
Or if they are not set, does it assume the first column is entity and second is date?

Reply:
No, they cannot be in any order, as mentioned in line 112.